src/Configuration/GrumPHP.php

```diff
@@ -171,6 +171,12 @@ public function getAsciiContentPath($resource)
             return null;
         }
 
+        // Deal with multiple ascii files by returning one at random.
+        if (is_array($paths[$resource])) {
+            shuffle($paths[$resource]);
+            $paths[$resource] = reset($paths[$resource]);
```

Comment:
You can just do a return of the reset here right away.

Reply:
I suppose there's already a precedent for an early return in this function, so 👍 .